src/python_framework_v01/nhd_network_utilities_v01.py

```diff
@@ -678,3 +678,57 @@ def read_waterbody_df(parm_file, lake_id_mask=None):
     if lake_id_mask:
         df1 = df1.loc[lake_id_mask, :]
     return df1
+
+def get_ql_from_wrf_hydro(input_files):
+    all_files = input_files
+    li = []
+
+    for filename in all_files:
+        ds = xr.open_dataset(filename)
+        df1 = ds.to_dataframe()
+
+        li.append(df1)
+
+    frame = pd.concat(li, axis=0, ignore_index=False)
+    mod = frame.reset_index()
+    ql = mod.pivot(index="station_id", columns="time", values="q_lateral")
+
```

Comment:
These should be independent... but because of how the restart file is missing indexes, we have to do a lot of munging of different things together.

Let's make the calls separate, just like we are doing. But let's pull the arguments completely apart. Eventually, these should be separate and the indexes should be self contained, and at that point, it will make sense. For now, we'll have some things happening multiple times, possibly.